pkg/applyconfigurations/testdata/cronjob/go.mod

```diff
@@ -0,0 +1,50 @@
+module sigs.k8s.io/controller-tools/pkg/applyconfigurations/testdata/cronjob
```

Comment:
Looking at all these dependencies. If I would have a separate module for the API to only have minimal dependencies, I probably wouldn't put the applicationconfigurations in the API folder.

Does the current configuration allow to put the applyconfigurations in an arbitrary folder? (not below the package for which they are generated)

Reply:
This would be a subpackage of the API, so Go is smart enough not to import the applyconfigurations subpackage if you aren't using it, however, it is a good point.

I was playing with this last week and I believe you can put a fully qualified package name in, will double check, and perhaps demonstrate that through a test somehow